src/renderer/lib/utils.ts

```diff
@@ -61,3 +62,12 @@ export const humanReadableBytes = (bytes: number): string => {
 
   return `0 ${sizes[0]}`;
 };
+
+export const isSubdir = (parentPath: string, dirPath: string) => {
+  const relativePath = path.relative(parentPath, dirPath);
+  return (
+    relativePath && // value is "" when paths are identical
```

Comment:
lets avoid throwing comments in line like this. rather put them together in one block before the return or create variables and add the comments there.